glide.yaml

```diff
@@ -47,6 +47,8 @@ import:
   version: master
 - package: github.com/openshift/service-serving-cert-signer
   version: release-3.10
+- package: github.com/openshift/source-to-image
+  version: release-3.10
```

Comment:
Why doesn't this float with master?  Why did this change? We already had this dep, didn't we?

Reply:
We're pinning until the release-3.10 branch is cut. s2i has a new feature in it that we don't want added to origin until 3.11.

Reply:
> We're pinning until the release-3.10 branch is cut. s2i has a new feature in it that we don't want added to origin until 3.11.

ok.